src/data/nat/factorization/prime_pow.lean

```diff
@@ -72,12 +97,8 @@ begin
     rintro q ⟨hq, hq'⟩,
     exact (nat.prime_dvd_prime_iff_eq hq hp).1 (hq.dvd_of_dvd_pow hq') },
   rintro ⟨p, ⟨hp, hn⟩, hq⟩,
-  -- Take care of the n = 0 case
   rcases eq_or_ne n 0 with rfl | hn₀,
```

Comment:
The comments still seem accurate, no need to delete them:
```suggestion
  -- Take care of the n = 0 case
  rcases eq_or_ne n 0 with rfl | hn₀,
```

Reply:
Ah, I thought we generally didn't keep explanatory comments in the middle of proofs, especially very simple ones like this that can easily be read off the code.